File: 프로그래머스/unrated/181861. 배열의 원소만큼 추가하기/배열의 원소만큼 추가하기.py
def solution(arr):
    # 이중 for문 사용. arr를 순회하며, 각 숫자 번씩 반복하며 각 숫자를 추가한다.
    answer = []
    
    for num in arr:
        for i in range(num):
            answer.append(num)
            
    return answer

File: 프로그래머스/unrated/181861. 배열의 원소만큼 추가하기/배열의 원소만큼 추가하기.js
function solution(arr) {
    // 방법 1 - 이중 for문
    const answer = []
    
    for (let num of arr) {
        for (let i = 0; i < num; i++) {
            answer.push(num)
        }
    }
    
    return answer
}